src/common/core/users/serialization.js

```diff
@@ -30,17 +30,440 @@ define(['common/util/assert', 'blob/BlobConfig'], function (ASSERT, BlobConfig)
     }
```

Comment:
While you're working on this file, can't you make it stateless?
I can see use-cases where exports are invoked before one has finished.